careers/technical-engineer.md

```diff
@@ -1,48 +1,78 @@
+- Bringing perspective from other teams or departments
 
-**Responsibilities:**
+**Mentorship:**
+
+A Staff Engineer is far more likely to change your company's long-term trajectory by growing other engineers than through personal heroics. Their effort here leads to a well-trained team with multiple experts in each domain.
+
+Effective mentorship is not a checkbox on the career ladder that can be satisfied by simply sharing advice and experience. Staff engineers build an ongoing relationship, understands the context of their mentee, and [sponsors](https://larahogan.me/blog/what-sponsorship-looks-like/) them.
+
+Concrete examples could include:
+
+- Owning onboarding and continuing education initiatives
+- No bus-factor-one concerns / build redundancy in systems and knowledge
+- Providing critical feedback on architectural design documents
+- Sponsorship
+
+**Exploration:**
 
-* Track metrics to ensure code quality standards
-* Advocate for improvements to product quality, security, and performance
-* Maintain a comprehensive technical knowledge of our entire software system
-* Assume responsibility for fixing problems / "Always Responsible Individual"
-* Participate in Tier 2 on-call rotations
-* Managing projects across multiple trimesters
+Our teams are already optimized to incrementally improve. The role of the staff engineer is to be *transformative*. A staff engineer does more than improve their part of the codebase; they solve ambiguous, important problems that the company's current systems are ill-shaped to address. They tackle impossible technical problems (high ambiguity, no in-house expertise, poorly defined). They identify and reduce existential risks. They look beyond their team and project to address organization-level problems (2-3 year growth, org health). They set the technical direction for a department.
+
+Concrete examples could include:
+
+- Reduce infrastructure costs by an order of magnitude
+- Identify a multi-region strategy that takes six months instead of three years
+- Addressing the sudden realization that your primary database only has three months of remaining disk space, and you can't upgrade to a larger size
+- Altering our CI/CD strategy to improve DORA metrics by 50% across the board
+- Identifying a software vendor that reduces churn and increases revenue by 10%
+
+**Be Glue:**
+
+Perhaps most difficult to define, Staff Engineers do [glue work](https://noidea.dog/glue). They see the important things that need done and make sure they happen (whether doing it themselves or finding the right person or process for the job).
+
+Concrete examples could include:
+
+- decreased onboarding time
+- got the team to agree on coding standards
+- reviewed designs and architecture decisions
+- made us build the right thing
+- made our customers happier
+- raised the organization's profile through public speaking/writing
+- simplified security and compliance concerns
 
 ## Principal Engineer
```

Comment:
I can't comment directly on the lines since they're outside the diff, but I've got some suggestions for this section.

1. 1st paragraph: add a comma after "QA":
> "(including but not limited to... engineering, QA, and data analysis)
2. 2nd paragraph: replace "is" with "are"
> "The problems that the Principal Engineer ~~is~~ are solving..."
3. 4th paragraph - this is very nit-picky, but I think "both" is a little confusing because one interpretation can be that there are two core projects, whereas it's actually referring to "Packback" and "side and experimental work". Maybe this is clearer?
> "...they are a prolific contributor to both Packback's core projects and their own external work."
4. 4th paragraph: I think adding some commas to this list would make it clearer:
> "When presented with a complex problem, process, or existing system, they..."
5. 4th paragrap: In the context of the words after this phrase, I think "leave" doesn't quite fit since it describes how the person should feel after the experience; IMO we should either swap it out with another word or tweak the subsequent phrase.

Swapping out "leave":
> "...working with this person should ~~leave~~ have team members feeling like..."

Tweaking the subsequent phrase:
> "...working with this person should leave team members feeling like they ~~are going to leave with~~ gained something significantly better than they came into.

6. 5th paragraph: I think repositioning the comma can make this sentence clearer.
> "They create architecture that shapes large parts of our business and ship complex projects, including many systems or major pieces of infrastructure."

7. 5th paragraph: The last sentence feels a little passive, what do you think of this?
> "They are the origin of multiple obvious and large technical contributions."

8. 6th paragraph: I think using semicolons instead of commas can make this sentence clearer.
> "... capable of setting short, medium, and long term strategic direction for parts of the technology stack; identifying areas of critical need based on future growth; and developing roadmaps to attack those problems."